i18n/en_US.csv

```diff
@@ -110,3 +110,10 @@
 "Customize Alma in your checkout","Customize Alma in your checkout"
 "Edit the text displayed when choosing the payment method in your checkout. It will adapt to the languages of your site.","Edit the text displayed when choosing the payment method in your checkout. It will adapt to the languages of your site."
 "Merge all payment methods","Merge all payment methods"
+"Payment upon trigger","Payment upon trigger"
```

Comment:
did you add this translations keys manually or is there a magento command to build translations files ?

Reply:
It's a manually add